source/exceptions.tex

```diff
@@ -317,6 +317,15 @@
 the selected constructor is odr-used\iref{basic.def.odr} and
 the destructor of \tcode{T} is potentially invoked\iref{class.dtor}.
 
+\pnum
+An exception is considered uncaught
```

Comment:
While we're at it, this looks like a definition and presumably should be formatted and indexed as one (exception, uncaught).

Reply:
Good point!  Added indexing, rebased, and force-pushed.